app/views/pages/tutorials/integrations/managing-sendgrid-partner-portal.liquid

```diff
@@ -0,0 +1,55 @@
+---
+converter: markdown
+metadata:
+  title: Managing Your SendGrid Integration through the Partner Portal
+  description: This guide will help you create SendGrid API keys using the Partner Portal API. 
+slug: tutorials/integrations/managing-sendgrid-partner-portal
+searchable: true
+---
+
+This guide will help you create SendGrid API keys using the Partner Portal API. 
+
+## Requirements
+To follow this tutorial, you should have access to SendGrid, and be familiar with the SendGrid API and know how to integrate SendGrid using API Call notifications. 
+
+* [SendGrid API Reference, Contacts API - Recipients: POST](https://sendgrid.api-docs.io/v3.0/contacts-api-recipients/add-recipients)
+* [Integrating SendGrid Using API Call Notifications](/tutorials/integrations/integrating-sendgrid-api-call-notifications)
+
+## Steps
+
+Managing SendGrid using the Partner Portal API is a three-step process:
+
+<div data-autosteps></div>
+
+### Step 1: Authenticate yourself
+
+Use the following command to authenticate yourself and receive an authorization token that you can use in the next requests: 
+
+```shell
+curl -X POST -H "Content-Type: application/json" -d '{"email":"user@example.com","password":"valid password"}' "https://partners.platform-os.com/api/authenticate"
+```
+
+### Step 2: Send POST request to create SendGrid account
+
+Send a POST request to create a SendGrid account including a webhook URL param. See the webhooks key in the [doc JSON](https://partners.platform-os.com/doc.json):
```

Comment:
Yeah, we can link to a generated version here, json isnt particularly human-friendly format :)
https://documentation.platformos.com/api-reference/partner-portal/api